docs/configuration.md

```diff
@@ -1365,6 +1366,13 @@ Apart from these, the following properties are also available, and may be useful
         </td>
     </tr>
     <tr>
+        <td><code>spark.ssl.keyStoreType</code></td>
+        <td>JKS</td>
+        <td>
+            The type of the key-store.
```

Comment:
I see many other places say `key-store` so keep consistent with them. should they be take out either?